services/static-webserver/client/source/class/osparc/data/Resources.js

```diff
@@ -103,6 +103,11 @@ qx.Class.define("osparc.data.Resources", {
             method: "POST",
             url: statics.API + "/projects/{studyId}:open"
           },
+          getWallet: {
+            useCache: false,
+            method: "GET",
+            url: statics.API + "/projects/{studyId}/wallet"
+          },
```

Comment:
Thanks! You are much faster than me :) We haven't discussed that yet, but I wanted to tell you that I think it is good that it doesn't live as a part of the project resource. The main reason is that most of the time when you work on a project you don't really need this information. Therefore we would make always unnecessary calls to ask for the wallet (as a part of it we are doing additional calls for example to check the access rights and so on..).

Reply:
Alles Klar